cmd/bridgec/internal/client.go

```diff
@@ -0,0 +1,119 @@
+package internal
+
+import (
+	"fmt"
+	"os"
+	"reflect"
+	"runtime"
+	"strings"
+
+	"github.com/spf13/cobra"
+	"github.com/threefoldtech/rivine/pkg/api"
+	"github.com/threefoldtech/rivine/pkg/cli"
+)
+
+// CommandLineClient extend for ERC20 commands
```

Comment:
Comment is wrong? I suppose you copy pasted it and forgot to edit. Please be clear and specific here. It will help others as well as your future self. It is ok to write in a couple of lines if that is what it takes.